java/arcs/sdk/android/demo/AndroidManifest.xml

```diff
@@ -0,0 +1,29 @@
+<!--
```

Comment:
Is `sdk` the right place for this? I think @csilvestrini had in mind that everything under `sdk` was DevEx oriented (the workstream area, not the runtime/env), but the purpose of this is to demo the new architecture. Either could fit but I thought to check.

Reply:
Maybe java/arcs/demo/android/... would be a better place for this. You're right Walter, `sdk` is supposed to be for the public developer-facing APIs. This demo would ideally be built on top of APIs exposed in the `sdk` package, rather than living in there.

Reply:
I think it's a good idea to place demo code as close as possible to the stuff it's demoing. And my understanding is that the Android SDK part is for developers to build new hosts with, i.e. I am not trying to demo how to use Arcs, which are covered in particles/tutorials already.

If java/arcs/sdk/android/demo is not a good place, how about java/arcs/android/demo? Although I still prefer this location as this app only demos how you can build hosts in Android with the Android SDK.

Reply:
java/arcs/android/demo sounds good to me

Reply:
OK moved.